components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c

```diff
@@ -721,26 +724,31 @@ psa_status_t psa_get_key_lifetime(psa_key_slot_t key,
     return ((psa_status_t) err_call);
 }
 
-psa_status_t psa_set_key_lifetime(psa_key_slot_t key,
-                                  psa_key_lifetime_t lifetime)
+psa_status_t psa_create_key(psa_key_lifetime_t lifetime,
+                            psa_key_id_t id,
+                            psa_key_handle_t *key_handle)
 {
     psa_error_t err_call;
-    psa_key_mng_ipc_t psa_key_mng_ipc = { 0, 0, 0 };
-    psa_key_mng_ipc.key = key;
-    psa_key_mng_ipc.type = 0;
-    psa_key_mng_ipc.func = PSA_SET_KEY_LIFETIME;
+    psa_key_mng_ipc_t psa_key_mng_ipc = { 0, 0, 0, 0 };
+    psa_key_mng_ipc.handle = *key_handle;
+    psa_key_mng_ipc.lifetime = lifetime;
+    psa_key_mng_ipc.func = PSA_CREATE_KEY;
     psa_handle_t handle = PSA_NULL_HANDLE;
     psa_invec_t in_vec[2] = {
-        { &psa_key_mng_ipc, sizeof(psa_key_mng_ipc_t) },
-        { &lifetime, sizeof(psa_key_lifetime_t) }
+        { &psa_key_mng_ipc, sizeof(psa_key_mng_ipc) },
+        { &id, sizeof(id) }
+    };
+    psa_outvec_t out_vec[1] = { {
```

Comment:
style:

   psa_outvec_t out_vec[1] = { 
         { key_handle, sizeof(*key_handle) }
    };

Reply:
this style is inconsistent throughout the whole file, see:
 https://github.com/ARMmbed/mbed-os/blob/dc54be2dc12b7d4b8a945211a61167be1377a9a0/components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c#L868-L872
https://github.com/ARMmbed/mbed-os/blob/dc54be2dc12b7d4b8a945211a61167be1377a9a0/components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c#L946-L950
 https://github.com/ARMmbed/mbed-os/blob/dc54be2dc12b7d4b8a945211a61167be1377a9a0/components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c#L1438-L1440